py/README.md

```diff
@@ -0,0 +1,76 @@
+# Deephaven Python Development
+
+## Requirements
+
+- Python >=3.8
+
+## Setting up Virtual Environment
+
+For any of the options below, you should first set up a virtual environment in Python. Once created, you must activate the virtual environment every time you open a shell. 
+
+```sh
+python3 -m venv .venv # Create. Only needed once
+source .venv/bin/activate # Activate the virtual environment
+deactivate # Exit virtual environment when done
+```
+
+**Note:**
+
+- This is not an exhaustive guide to managing Python environments.
+- Depending on your OS and how your PATH is set up, you may need to use `python3`, or a path to the explicit Python version you want to use.
+- You may set up a "permanent" virtual environment location.
+- You'll need to re-install the wheel whenever you make Python code changes that affect the wheel.
+- `pip` can be a pain if you are trying to (re-)install a wheel with the same version number as before.
+  - A `pip install --force-reinstall --no-deps "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"` may do the trick.
```

Comment:
Don't we also need to install the `py/server-embedded/.../deephaven_server` wheel? I've been doing that... but maybe I don't need to. :shrug:

Reply:
I've found it unnecessary if you're not using the pip installed version of DH (DHaaL I think we called it?)

This section was just moved from the jetty-app readme b/c it didn't make sense there IMO, but it does look like it was just a brain dump to get the info on the page